azure-pipelines/CI.yml

```diff
@@ -34,7 +34,7 @@ stages:
 
       - job: 'Windows'
         pool:
-          vmImage: 'vs2017-win2016'
+          vmImage: 'windows-2019'
```

Comment:
Why do we need this change? It would be better to stay on the old system for testing.

Reply:
There aren't that many dependency issues with different versions of windows. But the price using an old environment and not up to date compiler is way to high in my opinion.

Reply:
The thing is that you always test on the oldest supported system. Backward compatibility. Either we formally declare the Windows version no longer supported or we test for it. There are only these two ways we can take without a potential problem in the future. 
As declaring it no longer supported is not really necessary as Xournal++ works on it and we don't want to smaller our user base unnecessary we will have to test on it.